dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallUtils.java

```diff
@@ -734,12 +744,24 @@ public static Set<String> getFeatures(String serverXml, String xml, Set<String>
                 }
             }
         } catch (Exception e) {
-            logger.log(Level.FINE, Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_INVALID_SERVER_XML", xml, e.getMessage()));
+            if (opt == false) {
+                logger.log(Level.INFO, Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_INVALID_SERVER_XML", xml, e.getMessage()));
+            } else {
+                logger.log(Level.FINE, Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_INVALID_SERVER_XML", xml, e.getMessage()));
+            }
+
         }
         visitedServerXmls.add(isUrl ? serverXml : realServerXml.toString());
         for (String filepath : newLocations) {
             Path path = Paths.get(filepath);
-            features.addAll(getFeatures(path.toString(), path.getFileName().toString(), visitedServerXmls));
+            if (path.getFileName() != null) {
```

Comment:
Are we trying to check if the files exists here? Then we can do something like
`Files.exists(path)`

Reply:
Yes we are checking if the file exists, I will change it to file.exists.